docs/PROVISIONING_LIBVIRT.adoc

```diff
@@ -0,0 +1,186 @@
+
+FIXME:: the instructions are written _for now_ step by step and running locally on the libvirt host as root. This might/should change in the future but this is the current state of the implementation. Each sub-chapter is called after the playbook step constituting the end-to-end playbook.
+
+
+=== provision-instances
+
+- make sure `/dev/loopN` isn't mounted on `/var/www/html/installXXX`, and remove it from your `/etc/fstab` if you try multiple times with errors (something to FIXME).
+- copy and adapt the sample directory with files and inventory:
+* adapt the Libvirt specific parameters to make them compatible with your setup (especially the network)
+- export the variable `LIBVIRT_INV_VM_FILTER` to fit the libvirt names defined for your cluster's VMs, e.g. `export LIBVIRT_INV_VM_FILTER=^ocp_`.
+- if your network isn't persistent create it (see above).
+- make sure that `/tmp/authorized_keys` exists. FIXME: not sure yet for which use cases it is required, I just copy for now my own authorized keys.
+- call `ansible-playbook -i ../../inventory/sample.libvirt.example.com.d/inventory libvirt/provision.yml`.
++
+IMPORTANT: virt-install is only running synchronously because a virt-viewer UI is popping up. Close each virt-viewer once the corresponding installation has happened and not too long after.
++
+- identify the IP address of the infrastructure VM on which the route will run and start accordingly the separate dnsmasq responsible to do the wildcard DNS resolution (see above).
+- login into one of the new VMs and validate that DNS is working correctly:
+* `dig master.local` gives the correct IP address (same for all VMs)
+* `dig -x <master-ip>` works as well
+* `dig -x xxx.apps.local` gives the IP of the route/infranode.
+
+NOTE: up till now, I've worked as root to avoid complications. From here on, I'm working again as normal user on the control host.
+
+=== pre-install
+
+IMPORTANT: you need to have ssh-ed once to each node to make sure that their SSH-signature is already in your known_hosts file.
+
+Things to consider:
+
+- make sure the above preparations are still active (network, DNS, environment variables)
+- define the environment variables `RHSM_USER` and `RHSM_PASSWD` or use an activation key (TODO describe activation key / Satellite 6 approach).
++
+CAUTION: because there is no trace of OpenShift on the system, it is relatively sure that auto-attach will fail. Hence make sure `rhsm_pool` or `rhsm_pool_ids` are defined in the inventory (or on the command line).
+
+Then call `ansible-playbook -i ../../inventory/sample.libvirt.example.com.d/inventory/ pre-install.yml -e rhsm_pool='^{POOL_NAME}$'`.
```

Comment:
Looks like you're running this from a different directory than the last `ansible-playbook` command? Perhaps default to running commands from the root of the project?